<launch>

	<node pkg="vectornav" type="vn200_node" name="vectornav"  output="screen" > 
	  
	  <param name="serial_port" type="string" value="/dev/ttyUSB0" />
	  <param name="serial_baud" type="int"    value="115200" />
	  
	  <!-- Type: 0 None (Polling method), 19 IMU, 20 GPS, 22 INS -->
	  <param name="async_output_type" type="int"    value="0" />
	  <param name="async_output_rate" type="int"    value="100" />
	  
	  <param name="poll_rate"   type="int"    value="50" />
	  
	  <param name="imu/frame_id" type="string" value="LLA" />
	  <param name="gps/frame_id" type="string" value="LLA" />
	</node>	
	
</launch>


